production/loki-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -167,7 +167,7 @@ local utils = import 'mixin-utils/utils.libsonnet';
       tooltip: { sort: 2 },  // Sort descending.
     },
   containerCPUUsagePanel(title, containerName)::
-    self.CPUUsagePanel(title, 'container="%s"' % containerName),
+    self.CPUUsagePanel(title, 'container=~"%s"' % containerName),
```

Comment:
Curious why these changes were made? They seem unrelated to the purpose of the PR.
The compiled versions generally seem to not need the regex, and I'm concerned about containers like `query-scheduler` where `-` has a semantic meaning in regex.

Reply:
You're right! Fixing it.